docs/UserGuide.md

```diff
@@ -76,15 +76,15 @@ Format: `help`
 
 Adds a client to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS note/NOTE [t/TAG]…​`
```

Comment:
Email, address and note are optional for adding a new contact, so should they be wrapped with `[ ]`?
```suggestion
Format: `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [note/NOTE] [t/TAG]…​`
```